main.tf

```diff
@@ -0,0 +1,141 @@
+  tags = merge({
+    Name        = "${local.resource_prefix.value}-customer-master"
+    Environment = local.resource_prefix.value
+    }, {
+    git_commit           = "d68d2897add9bc2203a5ed0632a5cdd8ff8cefb0"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2020-06-16 14:46:24"
+    git_last_modified_by = "nimrodkor@gmail.com"
+    git_modifiers        = "nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "a7f01cc7-63c2-41a8-8555-6665e5e39a64"
+  })
+}
+
+resource "aws_s3_bucket" "financials" {
+  # bucket is not encrypted
+  # bucket does not have access logs
+  # bucket does not have versioning
+  bucket        = "${local.resource_prefix.value}-financials"
+  acl           = "private"
+  force_destroy = true
+  tags = merge({
+    Name        = "${local.resource_prefix.value}-financials"
+    Environment = local.resource_prefix.value
+    }, {
+    git_commit           = "d68d2897add9bc2203a5ed0632a5cdd8ff8cefb0"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2020-06-16 14:46:24"
+    git_last_modified_by = "nimrodkor@gmail.com"
+    git_modifiers        = "nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "0e012640-b597-4e5d-9378-d4b584aea913"
+  })
+
+}
+
+resource "aws_s3_bucket" "operations" {
+  # bucket is not encrypted
+  # bucket does not have access logs
+  bucket = "${local.resource_prefix.value}-operations"
+  acl    = "private"
+  versioning {
+    enabled = true
+  }
+  force_destroy = true
+  tags = merge({
+    Name        = "${local.resource_prefix.value}-operations"
+    Environment = local.resource_prefix.value
+    }, {
+    git_commit           = "d68d2897add9bc2203a5ed0632a5cdd8ff8cefb0"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2020-06-16 14:46:24"
+    git_last_modified_by = "nimrodkor@gmail.com"
+    git_modifiers        = "nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "29efcf7b-22a8-4bd6-8e14-1f55b3a2d743"
+  })
+}
+
+resource "aws_s3_bucket" "data_science" {
+  # bucket is not encrypted
+  bucket = "${local.resource_prefix.value}-data-science"
+  acl    = "private"
+  versioning {
+    enabled = true
+  }
+  logging {
+    target_bucket = "${aws_s3_bucket.logs.id}"
+    target_prefix = "log/"
+  }
+  force_destroy = true
+  tags = {
+    git_commit           = "d68d2897add9bc2203a5ed0632a5cdd8ff8cefb0"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2020-06-16 14:46:24"
+    git_last_modified_by = "nimrodkor@gmail.com"
+    git_modifiers        = "nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "9a7c8788-5655-4708-bbc3-64ead9847f64"
+  }
+}
+
+resource "aws_s3_bucket" "logs" {
```

Comment:
<details><summary>
<img src="https://bc-network-graph-icons.s3.us-west-2.amazonaws.com/prCommentsSeverityIcons/medium_new.svg" alt="MEDIUM" width="24px" align="center">&nbsp;&nbsp;<b style="color:black" >Ensure that an S3 bucket has a lifecycle configuration</b>
            <br>
            &nbsp;&nbsp;&nbsp;&nbsp;Resource: <a href="https://github.com/marynaKK/terragoat/blob/marynaKK-patch-1/main.tf#L113-L141">aws_s3_bucket.logs</a> | Bridgecrew ID: <code>BC_AWS_LOGGING_35</code> | Checkov ID: <a href="https://docs.prismacloud.io/en/enterprise-edition/policy-reference/aws-policies/aws-logging-policies/bc-aws-2-61">CKV2_AWS_61</a>
            <br>
            
</summary>
<h4>How to Fix</h4>

```tf
resource "aws_s3_bucket" "bucket" {
  bucket = "bucket"
  acl    = "private”

  lifecycle_rule {
    id      = "expire"
    status  = "Enabled"
    prefix  = "logs/"
    transition {
      days          = 30
      storage_class = "STANDARD_IA"
    }
    expiration {
      days = 90
    }
  }
}
```

<h4>Description</h4>

This policy detects whether an S3 bucket in AWS has a lifecycle configuration or not. A lifecycle configuration on an S3 bucket helps to manage objects so they can be automatically transitioned to other storage classes or expire after a certain period. This is crucial for cost effective data management and storage optimization.

</details>
<p><img src="https://bc-network-graph-icons.s3.us-west-2.amazonaws.com/prCommentsBotLogos/bc.svg" alt="Bridgecrew" width="18px">&nbsp;<small>Comment by Bridgecrew Bot</small></p>